src/components/AddItem.js

```diff
@@ -1,16 +1,19 @@
 import React, { useState } from 'react';
 import { useHistory } from 'react-router-dom';
 import { db } from '../lib/firebase';
+import Error from './Error';
 
 const initialFormState = {
   itemName: '',
+  cleanedUpItemName: '',
```

Comment:
I don't think that we are using `cleanedUpItemName` from state anywhere, so we can remove from the initial state. It's also state that can be derive from other state, so we wouldn't really need it. :-)